### DISCUSSION TITLE
How to make "find" to not select?

### DISCUSSION THREAD

**Discussion by quomat** [helix-editor/helix#6953] [Q&A] [answered]
I'm aware that this is blasphemous to the Helix community, as its core philosophy is about selection, but I find myself using "f□;d" combo all the time to delete single characters and "f□;i" to insert at a certain character... I thought that after using Helix for a month I would get used to it, but to no avail. I will gladly take some tips!

I tried to bind the f key to the combo [find,unselect] but it doesn't work because unselection happens before the second key of the find command... Is there a way to make it work?

Thank you for an amazing piece of software you all!

**Comment by guicanat** [comment]
Not that I would recommend it, but you could try something like this in your `config.toml`:
```
[keys.normal]
i = ["insert_mode", "collapse_selection"]
```

**Reply by gabydd** [reply]
I believe the collapse selection needs to be before insert mode

**Reply by quomat** [reply]
That could work, but I don't want to ruin Helix even further 😅

**Comment by pascalkuthe** [comment]
To insert a single character `tXa` should work. To delete a single char `tXa<del>` or `fXa<backspace>` would work altough if you dont want to insert afterwards (but stay in normal instead) then `fX;d` probably does work better

**Comment by kinxyo** [comment, marked as answer]
I mapped a custom keybind for this, so I can avoid 'un-selecting' every time I need to delete a character.

```toml
[keys.normal]
'\' =  ["collapse_selection", "keep_primary_selection", "delete_selection"]
```

Just use `fX\` now.

This is equivalent to `fXx` in Vim or `fX;d` in Helix.
